user-guide/Reference/Metrics/swarming_elements_benchmarks.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: swarming_bookings_benchmarks
+---
+
+# Swarming bookings benchmarks
+
+## Benchmarks
+
+| \# | Specification | Scope | Metric | Test Setup |
+| -- | ------------- | ----- | ------ | -------- |
+| 1 | Swarming 1 element | DMS | 0.591 s | DaaS cluster. |
+| 2 | Swarming 10 elements | DMS | 1.590  s | DaaS cluster. |
+| 3 | Swarming 100 elements | DMS | 9.670 s | DaaS cluster. |
+| 4 | Raw swarming time for 1 element | DMS | 0.154 s | DaaS cluster.<br /> Measured swarm action only, without time needed to start the element. |
+| 5 | Failover switch 100 elements | DMS | 50 s | Non-swarming system. Measured until Cube availability. |
+| 6 | Swarming 100 elements (Failover-like switchover) | DMS | 10 s | First element available after 2 seconds. |
```

Comment:
not really correct, i just used 2 seconds to not specify something hard. In reality they became available within 1 second (see swarm 1 element at the top).
Would just change this note here to say that the time was for all elements but the first element swarmed became available much sooner (as opposed to failover)